packages/core/src/parse/tokens-studio.ts

```diff
@@ -24,28 +24,14 @@ export function convertTokensStudioFormat(rawTokens: Record<string, unknown>): {
 
   function walk(node: unknown, path: string[]): void {
     if (!node || typeof node !== 'object') return;
-    for (const [k, v] of Object.entries(node)) {
+    for (const k in node) {
+      const v = (node as Record<string, unknown>)[k] as Record<string, unknown>;
       if (k.startsWith('$')) continue; // don’t scan meta properties like $themes or $metadata
 
       // token
       if (!!v && typeof v === 'object' && 'type' in v && 'value' in v) {
         const tokenID = [...path, k].join('.');
 
-        // resolve aliases
-        const tokenSet = path[0]!;
```

Comment:
TODO: figure out why this hack existed in the first place. Need to investigate whether Tokens Studio still does this—use pseudo-relative aliases or not (it’d be great if they didn’t any more)

Reply:
Figured it out lol